test/known_issues/test-crypto-authenticated-stream.js

```diff
@@ -0,0 +1,145 @@
+'use strict';
+// Refs: https://github.com/nodejs/node/issues/31733
+const common = require('../common');
+
+if (!common.hasCrypto)
+  common.skip('missing crypto');
```

Comment:
This is messing with the `known_issues` logic which expects tests to fail: https://ci.nodejs.org/job/node-test-commit-linux-containered/18011/nodes=ubuntu1804_sharedlibs_withoutssl_x64/testReport/junit/(root)/test/known_issues_test_crypto_authenticated_stream_2/

Reply:
Updated and squelched the linter warning.